pkg/metrics/client_go_adapter.go

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
NIT: The emoji should be ✨ 
(🌱 means that those changes would result in 0 impact on the end users - i.e. CI changes)